### ISSUE TITLE
process: not blocked by steb

### ISSUE THREAD

**Issue by Stebalien** [ipfs/go-ipfs#5778]
@ipfs/go-team,

I'm introducing a few things to try to not block people and to bring some of the good parts of working in the same office to our remote work environment.

* A "blocked by steb" label (suggested by david). Please apply this *and* ping me on IRC when something isn't going to move forward without my input. I wish github had a more flexible way to specify dependencies but... yeah.
* A calendly: https://calendly.com/stebalien. Please try to ping me on IRC first (calendly may say I'm free when I'm not) but I'm hoping this will make it easier to talk through problems synchronously.
* Tuesday "open office". That is, I will do my best to keep my schedule clear 9am-11pm (yay timezones...), Tuesday (PST). I'll also try to keep a zoom chat open at (https://protocol.zoom.us/my/stebalien).

Basically, I'm trying to open up as many communication channels as possible to make up for the fact that we don't just pass each other in the office every day.

**Comment by Stebalien** [comment]
(closing as I'm just using this issue as a mass email...)
